Mathlib/MeasureTheory/Covering/Vitali.lean

```diff
@@ -332,7 +332,7 @@ theorem exists_disjoint_covering_ae [MetricSpace α] [MeasurableSpace α] [Opens
   have M : (s \ ⋃ a ∈ u, B a) ∩
       ball x (R x) ⊆ ⋃ a : { a // a ∉ w }, closedBall (c a) (3 * r a) := by
     intro z hz
-    set k := ⋃ (a : v) (ha : a ∈ w), B a
+    set k := ⋃ (a : v) (_ha : a ∈ w), B a
```

Comment:
I think it's better with all of these to just use `_`.

Whenever I see a `_h` I inevitably start thinking "huh, I wonder why the unused argument linter made a mistake there, and the human needed to override it"...